README.md

```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # userstorage
 
-[![](https://img.shields.io/badge/pypi-v0.5.1-orange)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/python-2.7%20%7C%203.6%20%7C%203.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/badge/CI-passing-brightgreen)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
+[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/)  [![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) [![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml) [![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

Comment:
The state of the file at this commit looks good!
https://github.com/nirs/userstorage/blob/698d3a326ee5aebb0f57f474d8f8e90612dc5346/README.md

However the source is little bit messy:
```
[![](https://img.shields.io/pypi/v/userstorage)](https://pypi.org/project/userstorage/) [![](https://img.shields.io/pypi/pyversions/userstorage)](https://pypi.org/project/userstorage/)  [![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/) [![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml) [![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

It will be nicer if we put one badge per line - markdowns will render all badges in the same line:

```
[![](https://img.shields.io/pypi/v/userstorage)(https://pypi.org/project/userstorage/)
[![](https://img.shields.io/pypi/pyversions/userstorage)(https://pypi.org/project/userstorage/)
[![](https://img.shields.io/pypi/dm/userstorage)](https://pypi.org/project/userstorage/)
[![](https://github.com/nirs/userstorage/actions/workflows/ci.yml/badge.svg)](https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml)
[![](https://img.shields.io/github/license/nirs/userstorage)](https://pypi.org/project/userstorage/)
```

Some lines are too long but markdown does not support line breaks where we would
like to have them.

Another issue is linking the CI badge ci.yaml file:
https://github.com/nirs/userstorage/blob/master/.github/workflows/ci.yml

A more useful link is the ci workflow:
https://github.com/nirs/userstorage/actions/workflows/ci.yml

To prepare a final pull request, please:
1. Squash all commits to single commit 
   https://stackabuse.com/git-squash-multiple-commits-in-to-one-commit/
2. Put one badge per line
3. Fix ci badge target
4. Force push to your repo to update the pull request